x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum OpenModal {
+  None = 'None',
+  ConfirmModifyModal = 'Confirm Modify',
+  ConfirmResetModal = 'Confirm Reset',
+  ConfirmSaveModal = 'Confirm Save',
```

Comment:
Is this text that needs to be i18n'd?

Reply:
Good catch. I'm not sure why they were assigned strings like that, TBH. I updated them: https://github.com/elastic/kibana/pull/94629/commits/717b41a90e21debf450684626ccb6bfe68e825d5.